.eslintrc.json

```diff
@@ -22,6 +21,7 @@
         "anonymous": "always",
         "named": "never"
       }],
+      "object-curly-spacing": ["error", "never", {"objectsInObjects": true}],
```

Comment:
There's mixed use through the codebase but this seems to be our general pattern?